gslab_scons/tests/_test_helpers.py

```diff
@@ -46,18 +46,22 @@ def command_match(command, executable, which = None):
                          command)
 
     elif executable in ['r', 'R']:
-        # e.g. "R CMD BATCH --no-save '--args cl_arg' script.R script.log"
+        # e.g. "Rscript --no-save --no-restore --verbose script.R > script.log 2>&1"
```

Comment:
I think this is an example that would run in reality. Can you give me a real example of `args` to replace `args`?